tools/esp8266/app.cpp

```diff
@@ -4,13 +4,13 @@
 //-----------------------------------------------------------------------------
 
 #if defined(ESP32) && defined(F)
-  #undef F
-  #define F(sl) (sl)
```

Comment:
für die Zukunft fände ich es besser lesbar, wenn man #ifdef usw. einrücken könnte

Reply:
Du meinst statt:
#if defined(ESP32) && defined(F)
so:
#ifdefined(ESP32) && defined(F) ?

hmm ich bin kein Fan davon. Für mich gehört sich das schon getrennt.
Alleine durch #if Signalisiert es uns das es beim compilieren verarbeitet wird und nicht in der Laufzeit durchgefahren wird.
Wenn du es trotzdem so hand haben magst, kann ich es nochmal anpassen.

Reply:
nein statt
```
#if defined(ESP32) && defined(F)
#undef F
#define F(sl) (sl)
#endif
```

will ich es lieber so haben

```
#if defined(ESP32) && defined(F)
    #undef F
    #define F(sl) (sl)
#endif
```